### ISSUE TITLE
Show ose-secret data in view

### ISSUE THREAD

**Issue by TheWalkingLeek** [puzzle/cryptopus#384]
Cryptopus can also store Openshift/Kubernetes Secrets: https://docs.openshift.com/container-platform/3.11/dev_guide/secrets.html

to store and restore those secrets, ccli is used: https://github.com/puzzle/ccli/

Render the ose_secret data on the accounts-show page. This is a Base64 encoded String which results in a YAML text.
Parse this yaml and show only relevant data e.g 'password', 'type', ...
This is read-only!

# ToDo

- [x] Make sure there's at least one ose secret in encryptables fixtures
- [ ] make sure ose secret content is decrypted and transmitted to UI
- [ ] create show page for ose secret and format yaml
  - basically, ose secret is just a yaml file
